autobahn/wamp/interfaces.py

```diff
@@ -262,7 +262,10 @@ class ISession(object):
     @abc.abstractmethod
     def onConnect(self):
         """
-        Callback fired when the transport this session will run over has been established.
+        Callback fired when the transport this session will run over has
+        been established.
+
```

Comment:
Yes, that ability. The `onJoin` callback (should be named `on_join`, but that's a different story), can fire multiple times without the underyling transport to go away. Would that be ok with having all callback we allowed to return a deferred/future?